src/transformers/models/blip_2/configuration_blip_2.py

```diff
@@ -209,6 +209,7 @@ def __init__(
         position_embedding_type="absolute",
         cross_attention_frequency=2,
         encoder_hidden_size=1408,
+        qformer_text_input=False,
```

Comment:
Can you add a line in the docstring above to explain the aim of this arg? 🙏

Reply:
done